### PR TITLE
add `className` and `style` to more components + style updates + percy integration

### PR DESCRIPTION
**Pull request by chriddyp**
@alishobeiri - could you review this? best done commit-by-commit probably

### CODE REVIEW

**Comment by chriddyp** [comment]
Fixes https://github.com/plotly/dash-core-components/issues/57

File: src/components/Slider.react.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default class Slider extends Component {
                     }
                 }}
                 value={value}
-                {...omit('value', this.props)}
+                {...omit(['fireEvent', 'setProps', 'value'], this.props)}
```
Comment (alishobeiri) [inline src/components/Slider.react.js]: Maybe could add `updatemode` here as well

Comment (chriddyp) [inline src/components/Slider.react.js]: nice catch, thanks!

File: src/components/Checklist.react.js [hunk 2]
```diff
@@ -116,6 +97,39 @@ Checklist.propTypes = {
...
+    className: PropTypes.string,
+
+    /**
+     * The style of the container (div)
+     */
+    style: PropTypes.object,
+
+
+    /**
+     * The style of the <input> radio element
```
Comment (alishobeiri) [inline src/components/Checklist.react.js]: Should read checkbox instead of radio

File: src/components/RadioItems.react.js [hunk 3]
```diff
@@ -60,6 +62,44 @@ export default class RadioItems extends Component {
...
+     */
+    options: PropTypes.shape({
+        /**
+         * The checkbox's label
+         */
+        label: PropTypes.string,
+
+        /**
+         * The value of the checkbox. This value
+         * corresponds to the items specified in the
```
Comment (alishobeiri) [inline src/components/RadioItems.react.js]: Should read radioitem instead of checkboxx

File: src/components/RadioItems.react.js [hunk 4]
```diff
@@ -60,6 +62,44 @@ export default class RadioItems extends Component {
...
+         * `values` property.
+         */
+        value: PropTypes.string,
+
+        /**
+         * If true, this checkbox is disabled and can't be clicked on.
```
Comment (alishobeiri) [inline src/components/RadioItems.react.js]: radioitem instead of checkbox

**Comment by alishobeiri** [comment]
Otherwise I think looks good

**Comment by alishobeiri** [comment]
:dancer: :D

**Pull request merged**